Automattic/wp-calypso
# Stats: Adds an overview cards section on the Subscribers page

Related to #75928

## Proposed Changes

* This PR adds in the basic skeleton of a component to display the stats overview cards. This does not include real data or combining the styles. These will come as two follow-up PRs shortly. 

## Testing Instructions

* Open the Live branch.
* Visit Jetpack → Stats.
* Append ?flags=stats/subscribers-section to the URL and load.
* Visit the new Subscribers page.
* Confirm that 4 new cards for today, 30 days ago, 60 days ago, and 90 days ago are displayed
* As per the i1 designs, the change up/down display is not present on the 'today' card, only on the other three

**note on styling:***
This is using default styles borrowed from the highlights cards further up the page. Additional styling to get things right will come in a followup PR. 

![image](https://user-images.githubusercontent.com/30754158/235278308-54eb0e74-6256-4312-803b-b686b5170cac.png)


## Pre-merge Checklist
